app/models/organization_profile.rb

```diff
@@ -30,6 +31,7 @@ class OrganizationProfile < ApplicationRecord
   before_save :normalize_phone
 
   validates :phone_number, phone: {possible: true, allow_blank: true}
+  validates :about_us, presence: true
```

Comment:
can we for now not validate the presence of this? It will mean we have to provide more information when creating a new org, which is done by the console, and already has a fair bit of data required.

Reply:
Agreed with @kasugaijin here.

My rule about adding validations is to ask wither or not the record is valid if it is missing this the attribute in question. A profile could still be legit if it is missing a `about_us` section in my mind. In other cases maybe there are attributes that are so important that if its missing we shouldn't save those records. A common one is a User without a email if we rely on the email heavily.